### PR TITLE
test working

### LINKED ISSUE #8: TEST: Delete product
Write a new test in the `tests/product.py` module that verifies that a product can be deleted and not shown to users.

### PR DESCRIPTION
**Pull request by devinmgarcia**
Description of PR that completes issue here...

## Changes

- Added new test for delete starting at tests/product.py line 99
- Added 404 NOT FOUND return on views/product.py line 158

## Testing

Description of how to test code...

- [ ] Run command in the root directory:  python3 manage.py test tests -v 1
- [ ] Ensure all tests pass

## Related Issues

- Fixes #8

### CODE REVIEW

File: tests/product.py [hunk 1]
```diff
@@ -95,6 +96,19 @@ def test_get_all_products(self):
         self.assertEqual(response.status_code, status.HTTP_200_OK)
         self.assertEqual(len(json_response), 3)
 
-    # TODO: Delete product
+    def test_delete_product(self):
+        """
+        Ensure we can delete an existing product.
+        """
+        url = "/products/1"
+        self.test_create_product()
```
Comment (heymonicakay) [inline tests/product.py]: The `test_create_product` method is meant to test the create function. The existing Bangazon code also uses it to seed the test db, which is poor practice for a couple of reasons. 1. It violates the Single Responsibility Principle, and 2. it makes the code less 'DRY' (i.e. repetitive, sometimes referred to as 'WET' code).

Instead, there should be code written specifically for seeding the database with product data before each test. The `setUp` method runs before each test in a given test class, which makes it the ideal location for this sort of functionality.

Remove the `test_create_product` method from your `test_delete_a_product` method. Then, modify your `setUp` method to seed product data into the test db before each test. Ping me on Slack once you've pushed up changes, and then you'll be good to go!

**Review by heymonicakay — CHANGES_REQUESTED** [review]
Great work Devin! I left a comment requesting changes. Let me know if you have any questions about it!

File: tests/product.py (hunk 1 above)
Comment (devinmgarcia) [reply tests/product.py]: Let me know if the changes made satisfy the requirement. Thanks

**Review by heymonicakay — APPROVED** [review]
Very nice! Approved!

**Pull request merged**